doc/ref_arch/openstack/chapters/Terminology.md

```diff
@@ -0,0 +1,18 @@
+[<< Back](../../openstack)
+# Terminology
```

Comment:
All terminology should go to Glossary. we should not have different terminology per each spec.

Reply:
Yes why not! We can have a section in there for open srack. I think it will good to have a consolidated place for all terminologies to avoid duplication and inconsistency.

Reply:
Cool @rabiabdel makes sense - which workstream lead can we ask to create the framework and we'll create the OpenStack (RA-1) terms?

Reply:
Label changed to "won't address"; we can close this PR and associated issue once we have a place to add RA-1 specific terminology (with references to OpenStack and ETSI).

Reply:
@iangardner22 this can be added directly in: https://github.com/cntt-n/CNTT/blob/master/doc/tech/glossary.md

Reply:
OK - resolving conversation